markdown/util.py

```diff
@@ -317,12 +337,12 @@ def __iter__(self) -> Iterator[_T]:
         return iter([self._data[k] for k, p in self._priority])
 
     @overload
-    def __getitem__(self, key: str | int) -> _T:  # pragma: no cover
-        ...
+    def __getitem__(self, key: str | int) -> _T:
+        ...  # pragma: no cover
```

Comment:
I don't understand this change. The proper way to comment out a block of code is to comment on the parent line, not each individual line of the block. It just so happens that the block only contains one line here, but still. Why?

Reply:
I thought I saw it complain on my previous pull request, but I was mistaken and this can just be reverted.